app/uk/gov/hmrc/eusubsidycompliancefrontend/controllers/HelloWorldController.scala

```diff
@@ -32,11 +31,9 @@ import scala.concurrent.{ExecutionContext, Future}
 class HelloWorldController @Inject()(
   mcc: MessagesControllerComponents,
   helloWorldPage: HelloWorldPage,
```

Comment:
I think this file can just be removed

Reply:
yup I was thinking about it and then forgot 🤦‍♀️

Reply:
done 👍